CONTRIBUTING.md

```diff
@@ -137,6 +137,59 @@ clear what the branch brings to *pandas*. You can have many
 shiny-new-features and switch in between them using the git checkout
 command.
 
+### Creating a development environment
+
+An easy way to create a *pandas* development environment is as follows.
+First, install either Install Anaconda \<install-anaconda\> or
+Install miniconda \<install-miniconda\>.
+
+Tell `conda` to create a new environment, named `pandas_dev`, or any
+name you would like for this environment by running:
+
+    conda create -n pandas_dev pandas cython nose libpython
+
+For a python 3 environment
+
+    conda create -n pandas_dev python=3 pandas cython nose libpython
+
+This will create the new environment, and not touch any of your existing
+environment, nor any existing python installation. It will install all
+of the basic dependencies of pandas, as well as the development and
+testing tools. If you would like to install other dependencies, you can
```

Comment:
I would note that you'll still need to do an inplace install of pandas to test changes (per the directions in the next section)